### PR TITLE
Feature/form

### PR DESCRIPTION
**Pull request by melkamu12**
### Blog app: validations and model specs

In this PR, I added the controllers and view on Blog app element 

 ## Activities done

* [ ] Create a method called current_user in your ApplicationController to make current user data available to all controllers.
    - It will return the first user from the database.
    - Don't worry about proper authentication at this time. You will add it in the upcoming projects.
* [ ] Create forms to perform the following functions:
    - Creates a Post on behalf of the current_user (use the method that you created in your ApplicationController).
    - Create a comment on behalf of the current_user (use the method that you created in your ApplicationController).
* [ ] Allow Users to add likes to Posts.
# Checklist:
 * [ ] Follows the style guidelines of this project
 * [ ] There is no linters error in my code
 * [ ] follow gitflow

### CODE REVIEW

File: app/views/comments/new.html.erb
Comment (KDlamini) [inline app/views/comments/new.html.erb:1]: Well done with adding forms. The controllers, routes, and forms are implemented well 💯 

#### New Comments

- Kindly ensure that users are able to add comments on any post. Currently adding comments to a post using the form does not add the comment, the page just refresh but nothing is displayed.

**Review by KDlamini — CHANGES_REQUESTED** [review]
### Hi @melkamu12 🙋‍♀️,  

Great work so far! 💯 
You have done a great job adding forms to the Rails Blog app. 🎊 🥳
There are some issues that you still need to work on to improve your project according to the requirements. You are almost there! 💪🏾 

You have done a great job adding forms to the Rails Blog app. 🎊 🥳

## Highlights 🌞

✔️  Great PR title and description
✔️  Well-documented and professional README 
✔️  Forms in the views that accept input for creating posts and comments implemented
✔️  Controller actions for creating new posts, comments, and likes present
✔️  Respective routes for the new actions added

## Required Changes ♻️

**Kindly look through the comments below 👇 for required changes**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/comments/new.html.erb
Comment (melkamu12) [reply app/views/comments/new.html.erb:1]: Now the users are able to add comments on each post. thank you for your comment

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @melkamu12👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- Good job with forms. ✔️
- No linter errors. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**